ceremonies/src/lib.rs

```diff
@@ -415,12 +415,18 @@ pub mod pallet {
 		#[pallet::weight((1000, DispatchClass::Operational,))]
 		pub fn set_meetup_time_offset(
 			origin: OriginFor<T>,
-			meetup_time_offset: T::Moment,
+			meetup_time_offset: MeetupTimeOffsetType,
 		) -> DispatchResultWithPostInfo {
 			<T as pallet::Config>::CeremonyMaster::ensure_origin(origin)?;
 			if <encointer_scheduler::Pallet<T>>::current_phase() != CeremonyPhaseType::REGISTERING {
 				return Err(<Error<T>>::WrongPhaseForChangingMeetupTimeOffset.into())
 			}
+
+			// Meetup time offset needs to be in [-8h, 8h]
+			if meetup_time_offset.abs() > 8 * 3600 * 1000 {
```

Comment:
never hardcode arbitrary numbers! please make this a constant (doesn't need to be a configurable runtime constant, just a `const`)

Reply:
well, actually....forget it. in this case we should prevent that someone thinks this could be used to shift by more than 12h, which would break